src/tools/jetbrains-plugin.md

```diff
@@ -31,19 +30,20 @@ Then install the Dart plugin and tell it where to find the Dart SDK.
 
 Install a JetBrains IDE if you don't already have one.
 
-* <a href="https://www.jetbrains.com/idea/download/"
-  target="_blank" rel="noopener">Download IntelliJ IDEA</a> or,
-  to try out the latest Dart language features,
-  [install IntelliJ IDEA EAP.](https://confluence.jetbrains.com/display/IDEADEV/EAP)
-* Or <a href="https://www.jetbrains.com/products.html"
-  target="_blank" rel="noopener">find another JetBrains product.</a>
+* [Download IntelliJ IDEA.][IDEA]{:target="_blank" rel="noopener"}
+* To try out the latest Dart language features and IntelliJ functionality,
+  [install IntelliJ IDEA EAP.][IDEA EAP]{:target="_blank" rel="noopener"}
+* [Find another Jetbrains product.][Other]{:target="_blank" rel="noopener"}
```

Comment:
```suggestion
* [IntelliJ IDEA][IDEA]{:target="_blank" rel="noopener"}
* [IntelliJ IDEA EAP][IDEA EAP]{:target="_blank" rel="noopener"}
  (for early access to the latest Dart language features and IntelliJ functionality)
* [Another JetBrains product][Other]{:target="_blank" rel="noopener"}
```

Reply:
I wasn't a fan of the periods, so thanks for this!

Reply:
> I wasn't a fan of the periods, so thanks for this!

:) Unlike some other companies, we use periods for list items that are complete sentences (but not generally for other list items). I think we used to use periods for more kinds of list items, but that's just silly. https://developers.google.com/style/lists#capitalization-and-end-punctuation